### DISCUSSION TITLE
There is no folder wiki in texstudio-org repo, how can I change the wiki?

### DISCUSSION THREAD

**Discussion by octaeder** [texstudio-org/texstudio#2733] [Q&A] [answered]
I would like to change the link to the manual which still is `` http://texstudio.sourceforge.net/manual/current/usermanual_en.html``, see screenshot here:

![grafik](https://user-images.githubusercontent.com/102688820/205946567-e610397e-3ff2-4704-ae15-c7fe92bb82ac.png)
Is there a way to do so on my own?

Thx

**Comment by sunderme** [comment, marked as answer]
the wiki is locked as someone tried to put download links to external web sites.

**Reply by octaeder** [reply]
so I have to leave it to you to do the change. Thx.
